dataverse-persistence/src/main/resources/Bundle_pl.properties

```diff
@@ -1725,6 +1725,9 @@ file.unrestrict=Usu\u0144 zastrze\u017Cenie dost\u0119pu
 file.restricted.success=Pliki \u201D{0}\u201D zostan\u0105 zastrze\u017Cone po naci\u015Bni\u0119ciu na przycisk "Zapisz zmiany".
 file.download.header=Pobierz
 file.download.subset.header=Pobierz podzbi\u00F3r danych
+file.download.dataset=Pobierz Zbi\u00F3r danych
+file.download.dataset.original=Pobierz oryginalny zbi\u00F3r danych
+file.download.dataset.tab=Pobierz tabularyczny zbi\u00F3r danych
```

Comment:
I think that to be more consistent with what is already in place. We should name this buttons: "Format archiwalny (.tab)" and "Format pierwotny". In english "Archival Format" and "Original Format"